legacy/app/activity/activity-routes.js

```diff
@@ -0,0 +1,3 @@
+angular.module('ushahidi.activity.routes', [])
+
+.config(require('./activity-routes-lazyload.js'));
```

Comment:
You could add the whole "activity-routes-lazyload.js" to the config-parenthesis here instead of requiring the file, maybe that makes things more intuitive? In that case, you can remove the activity-routes-lazyload.js entirely.
Like this:

```
angular.module('ushahidi.activity.routes', [])
.config(
    [
        '$stateProvider',
        '$urlMatcherFactoryProvider',
    function (
        $stateProvider,
        $urlMatcherFactoryProvider
    ) {
        $urlMatcherFactoryProvider.strictMode(false);
        $stateProvider
        .state({
            name: 'activity',
            url: '/activity',
            controller: require('./activity.controller.js'),
            template: require('./activity.html'),
            lazyLoad: function ($transition$) {
                const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
                return System.import('/activity.js').then(mod => {
                    $ocLazyLoad.load(mod.ACTIVITY_MODULE);
                });
            }
        });
    }]
);
```

Reply:
Honestly, I thought of this. I also think it will make this part less complicated. I like the idea of using this method.